workflows/phylorthology.nf

```diff
@@ -279,50 +243,6 @@ workflow PHYLORTHOLOGY {
     ch_rem_gene_trees = IQTREE_REMAINING(ch_rem_trimmed_msas, []).phylogeny
     ch_versions = ch_versions.mix(IQTREE.out.versions)
 
-    // Collect these gene family trees and alignments;
-    // they will be used for unrooted species tree inference
-    // with Asteroid and downstream analysis with GeneRax and
-    // SpeciesRax
-
-    // Do this for both the core and remaining orthogroups
-    // First trees....
-    ch_core_gene_trees
-    .branch {
-        meta, phylogeny ->
-            trees  : phylogeny
-                return phylogeny
-    }
-    .collect()
-    .set { ch_all_core_trees }
-
-    ch_rem_gene_trees
-    .branch {
-        meta, phylogeny ->
-            trees  : phylogeny
-                return phylogeny
-    }
-    .collect()
-    .set { ch_all_rem_trees }
-
-    // Then the alignments.
-    ch_core_trimmed_msas
-    .branch {
-        meta, trimmed_msas ->
-            msas  : trimmed_msas
-                return trimmed_msas
-    }
-    .collect()
-    .set { ch_all_core_msas }
-
-    ch_rem_trimmed_msas
-    .branch {
-        meta, trimmed_msas ->
-            msas  : trimmed_msas
-                return trimmed_msas
-    }
-    .collect()
-    .set { ch_all_rem_msas }
-
```

Comment:
SO happy to have this gone. Was just ugly, and I always had a feeling that making all of those branches was actually useless/accomplished nothing?